pymc/distributions/multivariate.py

```diff
@@ -391,7 +357,7 @@ class MvStudentT(Continuous):
     rv_op = mv_studentt
 
     @classmethod
-    def dist(cls, nu, Sigma=None, mu=None, scale=None, tau=None, chol=None, lower=True, **kwargs):
+    def dist(cls, nu, *, Sigma=None, mu, scale=None, tau=None, chol=None, lower=True, **kwargs):
```

Comment:
You need to provide of the many alternatives, it raises later if none is given

Reply:
I think it would be nice if an unspecified (so None) mu would be equivalent to passing mu=zeros(n). And the same for the MvNormal?
Seems this wasn't happening before though...

Reply:
Mu now defaults to zero in both MvNormal and MvStudentT